src/libnnstreamer-edge/nnstreamer-edge-internal.c

```diff
@@ -987,21 +1023,53 @@ nns_edge_start (nns_edge_h edge_h, bool is_server)
     return NNS_EDGE_ERROR_INVALID_PARAMETER;
   }
 
+  eh->listener_ip = _get_ip_address ();
```

Comment:
Q: difference between ip and listener_ip. IMO it is same, so how about setting ip and port using the functions _get_ip_address() and nns_edge_get_available_port()?